tests/tracer/test_context.py

```diff
@@ -48,3 +48,30 @@ def test_eq(ctx1, ctx2):
 )
 def test_not_eq(ctx1, ctx2):
     assert ctx1 != ctx2
+
+
+def test_traceparent():
+    def validate_traceparent(context, sampled_expected):
+        version_hex, traceid_hex, spanid_hex, sampled_hex = context._traceparent.split("-")
+        assert version_hex == "00"
+
+        assert len(traceid_hex) == 32
+        assert traceid_hex == "{:032x}".format(context.trace_id)
+
+        assert len(spanid_hex) == 16
+        assert spanid_hex == "{:016x}".format(context.span_id)
```

Comment:
Why do we pull the `span_id` and `trace_id` directly off of the `context`, rather than the `span`? It looks like they should always be the same so maybe it doesn't matter? I'm just unsure if there's ever and instance where they could not be the same. If that's not an issue, please resolve.

Reply:
Sampling priority is set on the context and not on the span. Also the context object is used to propagate trace and span ids so this seemed like a logical place to add the traceparent. We could also move this property to the Span object. I don't have strong feelings either way.